api/v1beta1/auth_config_types.go

```diff
@@ -595,13 +596,16 @@ type Response struct {
 
 	Wristband *Response_Wristband   `json:"wristband,omitempty"`
 	JSON      *Response_DynamicJSON `json:"json,omitempty"`
+	Plain     *Response_Plain       `json:"plain,omitempty"`
 }
 
 func (r *Response) GetType() string {
 	if r.Wristband != nil {
 		return ResponseWristband
 	} else if r.JSON != nil {
 		return ResponseDynamicJSON
+	} else if r.Plain != nil {
```

Comment:
I reckon there's only one response type at a time, and we can't check it otherwise (?)